src/app/jscaip/tests/Encoder.spec.ts

```diff
@@ -73,4 +73,20 @@ describe('NumberEncoder', () => {
             NumberEncoderTestUtils.expectToBeCorrect(encoder, new T(false, 2, Player.ONE));
         });
     });
+    describe('disjunction', () => {
+        const encoder1: NumberEncoder<number> = NumberEncoder.numberEncoder(5);
+        const encoder2: NumberEncoder<boolean> = NumberEncoder.booleanEncoder;
+        const encoder: NumberEncoder<number | boolean> =
+            NumberEncoder.disjunction(encoder1, encoder2,
```

Comment:
all in one line or each line on its own ! 🦾

Reply:
fait !